Twig/MopaBootstrapInitializrTwigExtension.php

```diff
@@ -38,17 +48,13 @@ public function __construct(array $parameters)
     public function getGlobals()
     {
         $meta = $this->parameters['meta'];
-        $dns_prefetch = $this->parameters['dns_prefetch'];
-        $preconnect = $this->parameters['preconnect'];
         $google = $this->parameters['google'];
 
         // TODO: think about setting this default as kernel debug,
         // what about PROD env which does not need diagnostic mode and test
         $diagnostic_mode = $this->parameters['diagnostic_mode'];
 
         return array(
-            'dns_prefetch'      => $dns_prefetch,
-            'preconnect'        => $preconnect,
```

Comment:
We don't need to expose these variables as globals anymore as their values are now retrieved by the new functions defined in the extension.